docs/sources/tempo/release-notes/v2-2.md

```diff
@@ -0,0 +1,150 @@
+
+By streaming results to the client, you can start to look at traces matching your query before the entire query completes. This is particularly helpful for long-running queries; while the total time to complete the query is the same, you can start looking at your first matches before the full set of matched traces is returned.
+
+In addition to streaming partial results, we’ve merged a number of improvements to speed up TraceQL queries. Here are just a few:
+
+* Add support for query batching between frontend and queriers to improve throughput [PR [2677](https://github.com/grafana/tempo/pull/2677)]
+* Improve performance of TraceQL regex [PR [2484](https://github.com/grafana/tempo/pull/2484)]
+* Fully skip over Parquet row groups with no matches in the column dictionaries [PR [2676](https://github.com/grafana/tempo/pull/2676)]
+* New synchronous read mode for vParquet and vParquet2 [PRs [2165](https://github.com/grafana/tempo/pull/2165), [2535](https://github.com/grafana/tempo/pull/2535)]
+* Improved TraceQL throughput by asynchronously creating jobs. [PR [2530](https://github.com/grafana/tempo/pull/2530)]
+
+
+### Metrics summary API (experimental)
+
+Tempo has added an experimental API that returns RED metrics (span count, erroring span count, and latency information) for spans of `kind=server``` sent in the last hour, grouped by an attribute of your choice.
+For example, you could use this API to compare error rates of spans with different values of the `namespace` attribute.
+From here, you might see that spans from `namespace=A` have a significantly higher error rate than those from `namespace=B`.
+As another example, you could use this API to compare latencies of your spans broken down by the `region` attribute.
+From here, you might notice that spans from `region=North-America` have higher latencies than those from `region=Asia-Pacific`.
+
+This API is meant to enable ad-hoc analysis of your incoming spans; by segmenting your spans by attribute and looking for differences in RED metrics, you can more quickly isolate where problems like elevated error rates or higher latencies are coming from.
+
+Unlike RED metrics computed by Tempo’s metrics-generator, the values returned by this API are not persisted as time series. This has the advantage that you do not need to provide your own time series databases for storing and querying these metrics. It also allows you to compute RED metrics broken down by high cardinality attributes that would be too expensive to store in a time series database. Use the metrics generator if you want to store and visualize RED metrics over multi-hour or multi-day time ranges, or you want to alert on these metrics.
+
+To learn more about this API, refer to the[ metrics summary API documentation.](/docs/grafana-cloud/data-configuration/traces/metrics-summary-api/)
```

Comment:
does this need updated to point to a tempo page if you're saying this has been moved out of cloud docs?